File: How_Much_Cassava.cpp
#include <iostream>

using namespace std;

int main() {
    int curupira, boitata, boto, mapinguari, iara;
    
    // ৫ জন অতিথির অংশের সংখ্যা ইনপুট নেওয়া
    if (cin >> curupira >> boitata >> boto >> mapinguari >> iara) {
        
        // মোট গ্রামের হিসাব (অতিথিদের খাবার + ডোনা চিকার ২২৫ গ্রাম)
        int total_cassava = (curupira * 300) + 
                            (boitata * 1500) + 
                            (boto * 600) + 
                            (mapinguari * 1000) + 
                            (iara * 150) + 
                            225;

        cout << total_cassava << "\n";
    }

    return 0;
}

File: How_Much_Cassava.py
import sys

def solve():
    input_data = sys.stdin.read().split()
    if not input_data:
        return

    curupira = int(input_data[0])
    boitata = int(input_data[1])
    boto = int(input_data[2])
    mapinguari = int(input_data[3])
    iara = int(input_data[4])

    total = (curupira * 300) + (boitata * 1500) + (boto * 600) + (mapinguari * 1000) + (iara * 150) + 225
    print(total)

if __name__ == "__main__":
    solve()